scripts/dev/check_for_malformed_enums.py

```diff
@@ -130,7 +130,7 @@ def process_enum_str(input_str: str, file_name: str, line_no: int, print_errors:
         # exceptions listed by <FILE>:<ENUM NAME>
         exceptions = [
             "HVACInterfaceManager.cc:UpdateType",
-            "IdfParser.hh:Token"
+            "IdfParser.hh:Token",
```

Comment:
In development I had added an extra exception, so this line needed a comma.  I ended up taking out the extra exception, but the comma stayed.  It's not hurting anything.